specs/latest/2.0/index.html

```diff
@@ -415,6 +415,7 @@ <h2 class="no-toc">Table of contents</h2>
 typedef (Int32Array or sequence&lt;GLint&gt;) ClearBufferIVSource;
 typedef (Uint32Array or sequence&lt;GLuint&gt;) ClearBufferUIVSource;
 typedef (Float32Array or sequence&lt;GLfloat&gt;) ClearBufferFVSource;
+typedef (ArrayBuffer or SharedArrayBuffer) ByteBuffer;
```

Comment:
There's already a BufferDataSource typedef in the WebGL 1.0 spec which covers uploads from ArrayBuffer or ArrayBufferView. That should be the typedef used throughout this PR -- there's no need to restrict these new overloads to ArrayBuffer and SharedArrayBuffer. Also, I think SharedArrayBuffer support should be added in a separate pull request.